platforms/chibios/converters/promicro_to_michi/converter.mk

```diff
@@ -0,0 +1,9 @@
+# Boardsource Michi MCU settings for converting AVR projects
```

Comment:
> Boardsource

Don't think this is meant to be here?
In fact, looking at [this picture](https://raw.githubusercontent.com/ci-bus/michi/master/conjunto3d.jpeg), it doesn't seem like Michi matches the Boardsource Blok pinout at all (which is what has been to copied in [`platforms/chibios/converters/promicro_to_michi/_pin_defs.h`](https://github.com/qmk/qmk_firmware/pull/19149/files#diff-73270eea75036ffb8f0c71e00d549389f38397d4e95ebb0dfa652e184060bfe6)).
For example Blok uses GPIO16/17 for the I2C pins that are broken out. Comparing to image of Michi, it uses GPIO2/3.

Reply:
I have used the Blok config like template, now Boardsource word has been removed, thanks 👍

Reply:
Hello @waffle87 is this change for you? please if it is ok resolve the conversation, very thanks

Reply:
Yeah, PR LGTM